src/components/line/sparkline/chartContentWrapper.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import { useEmpty, useInitialLoading } from "@/components/provider"
+import ChartContainer from "@/components/chartContainer"
+import Skeleton from "@/components/line/skeleton"
+import { Container } from "@/components/line/chartContentWrapper"
+
+const ChartContentWrapper = () => {
+  const initialLoading = useInitialLoading()
+  const empty = useEmpty()
+
+  return (
```

Comment:
Do we have keyboard listeners with sparklines? if yes, we need to pass cursor styling here as well.